tests/rss/test_rss.py

```diff
@@ -43,18 +48,77 @@ def test_gap_rss(test_dir, memory_jobstore, clean_dir):
         if i is not None:
             output_filter.append(i)
    
-    assert len(output_filter) == 1
+    assert len(output_filter) == 2
+
+    dir = Path('.')
+    path_to_job_files = list(dir.glob("job*"))
+    for path in path_to_job_files:
+        shutil.rmtree(path)
 
 
+def test_gap_rss_multi_jobs(test_dir, memory_jobstore):
+    from ase.units import GPa
+    np.random.seed(42)
+    test_files_dir = test_dir / "data/rss.extxyz"
+    atoms = read(test_files_dir, index="0:2:1")
+    structures = [AseAtomsAdaptor.get_structure(atom) for atom in atoms]
+    mlip_path = test_dir / "fitting/GAP"
+
+    job = do_rss_multi_node(mlip_type='GAP',
+                iteration_index='0',
+                mlip_path=mlip_path,
+                structure=structures,
+                scalar_pressure_method='exp',
+                scalar_exp_pressure=100,
+                scalar_pressure_exponential_width=0.2,
+                scalar_pressure_low=0,
+                scalar_pressure_high=50,
+                max_steps=1000,
+                force_tol=0.01,
+                stress_tol=0.0001,
+                hookean_repul=False,
+                write_traj=True,
+                num_processes_rss=4,
+                device="cpu",
+                isolated_atom_energies={14: -0.84696938},
+                num_groups=2,)
+    
+    response = run_locally(
+        job,
+        create_folders=True,
+        ensure_success=True,
+        store=memory_jobstore
+    )
+
+    output = job.output.resolve(memory_jobstore)
+    output_filter = []
+    for i in output:
+        if i:
+            output_filter.append(i)
+   
+    assert len(output_filter) == 2
+
+    ats = read(output_filter[0][0])
+
+    enthalpy_pseudo = ats.info["enthalpy"]
+    enthalpy_cal = ats.get_potential_energy() + ats.info["RSS_applied_pressure"]*GPa*ats.get_volume()
+    
+    assert round(enthalpy_pseudo,3) == round(enthalpy_cal,3)
+
+    dir = Path('.')
+    path_to_job_files = list(dir.glob("job*"))
+    for path in path_to_job_files:
+        shutil.rmtree(path)
+
 
 # def test_jace_rss(test_dir, memory_jobstore):
```

Comment:
Why is this test commented out?

Reply:
This test requires the python-lammps interface, meaning that we need to have LAMMPS installed to test this module. The current setup does not include LAMMPS. Although I have successfully tested it on our cluster, I am considering waiting to test it together once we introduce the MD module. Do you have any better suggestions?

Reply:
It is open source right? Then we can include it in our CI setup as well. Just let us know the version you have it tested on. I see there is option to install via conda as well. Did you use the one from Conda?

Reply:
Yes, it is. But I use the standard compilation method by cmake (see https://docs.lammps.org/Build_cmake.html). I am not sure if we can use conda to install lammps and related packages (i.e., pace and python here).

Reply:
There is the Conda option: https://docs.lammps.org/Install_conda.html. However, we need to ensure that it includes the PACE package (see https://docs.lammps.org/Build_extras.html#ml-pace).

Reply:
Thanks, building with cmake should also be possible. I Will try to add it in another PR today or in next days in our docker image here and then this test could also be enabled. Just let me know the version you have on your systems.